src/event.h

```diff
@@ -47,10 +47,12 @@ enum class event_type : int {
     character_wakes_up,
     character_wields_item,
     character_wears_item,
+    consumes_alcohol,
```

Comment:
Consuming alcohol is an unnecessarily specific event.  It would be better to have a `character_consumes_item` event, and an `event_transformation_function` that converts an item into its constituent materials.  Then you can construct the `consumes_alcohol` in json, but it would also work for defining e.g. a vegan or vegetarian conduct (which we're certainly going to want).